### PR TITLE
Update DG appendix

### LINKED ISSUE #220: Update DG - Appendix
Exclude Use case and User stories

### LINKED ISSUE #223: Update documentation - only Java 11
> Java 11 not accepted

### PR DESCRIPTION
**Pull request by chrystalquek**
Close #223 (Java 11)
Close #220 (DG Appendix)

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -705,28 +703,26 @@ Priorities:
...
+1. The computer should have sufficient RAM and CPU power to run the JVM smoothly.
 
-1. Should work on any mainstream OS as long as it has Java 11 or above installed.
-1. Should be able to hold up to 1000 contacts without any negative effects.
-1. Should be able to hold up to 1000 deliverables without any negative effects.
-1. Should be able to schedule meetings up to 2 years in advance.
-1. A user with above average typing speed for regular English text (i.e. not code, not system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
-
-*{More to be added}*
+## **Appendix E: Glossary**
```
Comment (shadowezz) [inline docs/DeveloperGuide.md:716]: Capitalise first word of glossary below

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -705,28 +703,26 @@ Priorities:
...
 
-### Glossary
+* **Mainstream OS**: Windows, Unix, OS-X.
+* **OCP**: Overall Completion Percentage. It is a piechart showing the project's completion status, found on the left panel of the Dashboard.
+* **Mode**: an option allowing a change in the method of operation of the app. The app can be in dashboard, deliverable, meeting and contact mode.
```
Comment (shadowezz) [inline docs/DeveloperGuide.md:720]: Sounds weird but idk what to change to

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -705,28 +703,26 @@ Priorities:
...
+* **Deliverable**: a good or service produced as a product of a development process in a project.
```
[the remark this answers is not in the document]
Comment (chrystalquek) [reply docs/DeveloperGuide.md:721]: or product development process?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (chrystalquek) [reply docs/DeveloperGuide.md:716]: where sorry

File: docs/DeveloperGuide.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (shadowezz) [reply docs/DeveloperGuide.md:721]: yea sure

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (shadowezz) [reply docs/DeveloperGuide.md:716]: like for each of the term below the explanation part

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (shadowezz) [reply docs/DeveloperGuide.md:720]: Maybe change the first sentence to "The state of the application that affects how each command will be executed."

**Review by shadowezz — APPROVED** [review]
LGTM

**Pull request merged**